requirements.txt

```diff
@@ -1,8 +1,10 @@
+--index-url https://pypi.devinfra.sentry.io/simple
+
 black==22.6.0
 blinker==1.5
 click==8.1.3
 clickhouse-driver==0.2.4
-confluent-kafka==1.8.2
```

Comment:
I had to additionally update confluent-kafka and arroyo because I didn't build confluent-kafka<1.9.2 for pypi.  previous versions have various trouble building on the 4 supported platform/arch 2-ples